tests/unit/main.cpp

```diff
@@ -42,15 +43,15 @@ int main(int argc, char* argv[]) {
   pe.insert("LANG", "en");
   pe.insert("LANGUAGE", "en");
 
-  QCoreApplication a(argc, argv);
+  QApplication app(argc, argv);
```

Comment:
Had to make this change after importing the systray modules. Is this an issue @bakulf?

Reply:
Hm, I think this is the culprit in breaking the Linux tests :thinking:

Reply:
Addressed in [eef9b69](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4327/commits/eef9b69cf5c213755eb842d591752a6150ae19eb)